### PR TITLE
Update support policy to reflect changes

### PR DESCRIPTION
**Pull request by therealmitchconnors**
I don't love the phrasing for Minor releases... suggestions welcome.

### CODE REVIEW

File: content/en/docs/releases/supported-releases/index.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ current `<minor>` release. A patch is usually a small change relative to the `<m
 | 1.13            | Yes                  | February 11, 2022 | ~October 2022 (Expected) | 1.20, 1.21, 1.22, 1.23        | 1.16, 1.17, 1.18, 1.19    |
 | 1.12            | Yes                  | November 18, 2021 | ~June 2022 (Expected)    | 1.19, 1.20, 1.21, 1.22        | 1.16, 1.17, 1.18          |
 | 1.11            | Yes                  | August 12, 2021   | ~Mar 2022 (Expected)     | 1.18, 1.19, 1.20, 1.21, 1.22  | 1.16, 1.17                |
-| 1.10            | No                   | May 18, 2021      | Dec 30, 2021             | 1.18, 1.19, 1.20, 1.21        | 1.16, 1.17, 1.22          |
+| 1.10            | No                   | May 18, 2021      | Jan 7, 2022             | 1.18, 1.19, 1.20, 1.21        | 1.16, 1.17, 1.22          |
```
Comment (ericvn) [inline content/en/docs/releases/supported-releases/index.md:60]:
```suggestion
| 1.10            | No                   | May 18, 2021      | Jan 7, 2022              | 1.18, 1.19, 1.20, 1.21        | 1.16, 1.17, 1.22          |
```

File: content/en/docs/releases/supported-releases/index.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ assistance. Separately, 3rd parties and partners may offer longer-term support s
...
 |-------------------|-----------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------|
 | Development Build | No support                                                                                                            | Dangerous, may not be fully reliable. Useful to experiment with.                                               |
-| Minor Release     | Support is provided until 3 months after the next minor release                                                       | Safe to deploy in production. Users are encouraged to upgrade to these releases as soon as possible.           |
+| Minor Release     | Support is provided until 6 weeks after the second subsequent minor release                                                       | Safe to deploy in production. Users are encouraged to upgrade to these releases as soon as possible.           |
```
Comment (ericvn) [inline content/en/docs/releases/supported-releases/index.md:33]: Not sure of the spacing to offer a suggestion here.

**Comment by jwendell** [comment]
@therealmitchconnors ping

**Comment by craigbox** [comment]
> "Support is provided until 6 weeks after the second subsequent minor release"

It's longer, but I think it would be easier to parse something like _"Support is provided until two further minor releases have been published, and then for six more weeks following that"_ ?

**Review by ericvn — APPROVED** [review]

**Pull request merged**